README.md

```diff
@@ -46,30 +45,43 @@ Here are the steps to quickly get started with this project:
     pre-commit install
     ```
 
-4. **Generate a Personal Access Token (PAT) for GitHub:** Ensure it only has read access.
+4. Set up [GitHub][0] and/or [GitLab][1] scraper(s).
 
-5. **Set your GitHub username and PAT as environment variables:** Replace `<user>` with your GitHub username and `<pat>` with your generated PAT, then run the following commands:
+5. **Start the collector:** Finally, initiate the program:
 
     ```bash
-    export GH_USER=<user>
-    export GH_PAT=<pat>
+    make run
     ```
 
-6. **Start the collector:** Finally, initiate the program:
+### Configure GitHub Scraper
```

Comment:
On both the configuration of the GH & GL scrapers, let's link them to the comments in the config.yaml file showing how to uncomment the extensions and update the pipelines.

Reply:
I updated the README to cover this how I think it should be but any input would be apprecaited!